libcnb-cargo/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## [Unreleased]
+- `cargo libcnb package` now allows multiple targets, as long as there is only one binary target. ([#282](https://github.com/Malax/libcnb.rs/pull/282))
```

Comment:
It's unlikely others are using libcnb-cargo as a library yet, but moving forwards we should probably mention all breaking changes in the changelog (with our open source maintainer hats on :-)).

```suggestion
- `cargo libcnb package` now allows multiple targets, as long as there is only one binary target. To support this change, the `BuildError` variants `NoTargetsFound` and `MultipleTargetsFound` have been replaced by `NoBinTargetsFound` and `MultipleBinTargetsFound` respectively. ([#282](https://github.com/Malax/libcnb.rs/pull/282))
```